### ISSUE TITLE
Call for Backup Robot can pull PU items in vanilla gamemodes

### ISSUE THREAD

**Issue by JuniorUdale** [TerryCavanagh/diceydungeons.com#1188]
Saw a streamer playing Jester 1 use Robot, and get Iron Armor? from it. Should probably exclude PU variants of items when you're not using PU rules

**Comment by TerryCavanagh** [comment]
Yep, good point! This is something I can easily do now with the modding changes for modding v1.0.
